src/transfer.jl

```diff
@@ -120,36 +146,29 @@ end
 Return the Fourier coefficients for `ψk · e^{i q·r}` in the basis of `kpt_out`, where `ψk`
 is defined on a basis `kpt_in`.
 """
-function multiply_by_expiqr(basis, kpt_in, q, ψk; transfer_fn=transfer_blochwave_kpt)
-    kcoords = getfield.(basis.kpoints, :coordinate)
+function multiply_by_expiqr(basis, kpt_in, q, ψk)
+    indices_kpints_σ = findall(e -> e.:spin == kpt_in.spin, basis.kpoints)
+    kcoords = getfield.(basis.kpoints, :coordinate)[indices_kpints_σ]
     shifted_kcoord = kpt_in.coordinate .+ q  # coordinate of ``k``-point in ℝ
-    (index, ΔG) = find_equivalent_kpt(kcoords, shifted_kcoord)
-    kpt_out = basis.kpoints[index]
-    return transfer_fn(ψk, basis, kpt_in, kpt_out, ΔG)
+    index, ΔG = find_equivalent_kpt(kcoords, shifted_kcoord)
```

Comment:
have this function take kpoints, shifted_kcoord and sigma (both kcoords and sigma are needed to identify kpoints)